File: Solutions/Java/Print_first_n_Fibonacci_Numbers.java
class Solution
{
    //Function to return list containing first n fibonacci numbers.
    public static long[] printFibb(int n) 
    {
        //Your code here
        long []fib = new long[n];
        
        fib[0] = 1;
        if(n == 1){
            return fib;
        }
        
        fib[1] = 1;
        
        for(int i=2; i<n; i++){
            fib[i] = fib[i-1] + fib[i-2];
        }
        
        return fib;
    }
}

File: Solutions/Python/Print_first_n_Fibonacci_Numbers.py
class Solution:
    #Function to return list containing first n fibonacci numbers.
    def printFibb(self,n):
        # your code here
        ans=[None]*n
        if n>=1:
            ans[0]=1
        if n>=2:
            ans[1]=1
        for i in range(2,n):
            ans[i]=ans[i-1]+ans[i-2]
        return ans
